tfjs-backend-webgpu/src/shader_preprocessor.ts

```diff
@@ -244,6 +294,14 @@ const commonSnippet = `
     return dot(coords, vec4<i32>(
         shape.y * shape.z * shape.w, shape.z * shape.w, shape.w, 1));
   }
+  fn getIndexFromCoords5D(coords : vec5, shape : vec5) -> i32 {
+    let currentShape: vec5 = vec5(shape.y * shape.z * shape.w * shape.u, shape.z * shape.w * shape.u, shape.w * shape.u, shape.u, 1);
```

Comment:
nit: Is `currentShape` -> `shapeStrides` better?

Reply:
Done.